components/tooling/glean-gradle-plugin/build.gradle

```diff
@@ -55,6 +55,15 @@ publishing {
     }
 }
 
+task renameSnapshotArtifacts(type: Exec) {
```

Comment:
Note to self: We need to copy [this function](https://github.com/mozilla-mobile/android-components/blob/7a3ba5719edec5cc49f7b3ef0c11faf336bda572/publish.gradle#L67) over, because  `tooling-glean-gradle` doesn't publish an aar, so it cannot reuse it.

We may want to factorize this behavior once we support more jars. In the meantime, I'm fine keeping it this way 🙂